01-Subscription-Security/Readme.md

```diff
@@ -186,8 +186,10 @@ Set-AzSDKSubscriptionSecurity -SubscriptionId <subscriptionId> -SecurityContactE
 |SecurityContactEmails 	|Comma-separated list of emails (e.g., 'abc@microsoft.com, def.microsoft.com')	for contact preference|
 |SecurityPhoneNumber 	|Single phone number (e.g., '425-882-8080' or '+91-98765-43210' or '+1-425-882-8080')	for contact preference|
 
-When the script starts, it removes existing/previously configured AzSDK artifacts in the subscription such 
-as alerts, RBAC, ARM policies, etc. It also *overwrites* the contact emails and contact phone previously set in Azure Security Center.
+**Note**: This command *overwrites* the contact emails and contact phone previously set in Azure Security Center.
+
```

Comment:
-phone number should be there instead of  contact phone
-configured can be used instead of set

Reply:
Resolved.